misoc/cores/vexriscv/core.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from migen import *
+
+from misoc.interconnect import wishbone
+
+
+class VexRiscv(Module):
+    def __init__(self, platform, cpu_reset_address):
+        self.ibus = i = wishbone.Interface()
+        self.dbus = d = wishbone.Interface()
+
+        self.interrupt = Signal(32)
+        self.timer_interrupt = Signal(1)
+
+        self.external_reset_vector = Signal(32)
```

Comment:
This doesn't need to have a signal, and doesn't need to be a property of the object. Just do ``i_externalResetVector=cpu_reset_address``.